helmfile.d/charts/gatekeeper/templates/policies/tests/reject-local-storage-empty-dir.rego

```diff
@@ -0,0 +1,393 @@
+package test.k8srejectlocalstorageemptydir
+
+import data.k8srejectlocalstorageemptydir
+
+#
+# Help functions
+#
+
+generate_resources(kind, volumeAnnotation, podAnnotation, annotations, volumes) = obj {
+    kind == "Pod"
+	obj := {
+		"parameters": {
+			"volumeAnnotation": volumeAnnotation,
+			"podAnnotation": podAnnotation
+		},
+		"review": {
+			"object": {
+				"kind": "Pod",
+				"metadata": {
+					"annotations": annotations
+				},
+				"spec": {
+					"volumes": volumes
+				}
+			}
+		}
+	}
+}
+generate_resources(kind, volumeAnnotation, podAnnotation, annotations, volumes) = obj {
+    allowed_kinds := [
+        "Deployment",
+        "StatefulSet",
+        "DaemonSet",
+        "ReplicaSet",
+        "Job",
+        "ReplicationController"
+    ]
+	kind == allowed_kinds[_]
+	obj := {
+		"parameters": {
+			"volumeAnnotation": volumeAnnotation,
+			"podAnnotation": podAnnotation
+		},
+		"review": {
+			"object": {
+				"kind": kind,
+				"spec": {
+					"template": {
+						"metadata": {
+							"annotations": annotations
+						},
+						"spec": {
+							"volumes": volumes
+						}
+					}
+				}
+			}
+		}
+	}
+}
+generate_resources(kind, volumeAnnotation, podAnnotation, annotations, volumes) = obj {
+	kind == "CronJob"
+	obj := {
+		"parameters": {
+			"volumeAnnotation": volumeAnnotation,
+			"podAnnotation": podAnnotation
+		},
+		"review": {
+			"object": {
+				"kind": "CronJob",
+				"spec": {
+					"jobTemplate": {
+						"spec": {
+							"template": {
+								"metadata": {
+									"annotations": annotations
+								},
+								"spec": {
+									"volumes": volumes
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+}
+
+kinds := [
+	"Pod",
+	"Deployment",
+	"StatefulSet",
+	"DaemonSet",
+	"ReplicaSet",
+	"Job",
+	"ReplicationController",
+	"CronJob"
+]
+
+#
+# Tests
+#
+
+test_no_volume {
+    res = [test | test = count(k8srejectlocalstorageemptydir.violation) == 0 with input as generate_resources(
+		kinds[_],
+        "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes",
+		"cluster-autoscaler.kubernetes.io/safe-to-evict",
+        {},
```

Comment:
Nit: Don't mix space and tabs. Seems like there's a mix in this file.
```suggestion
        "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes",
        "cluster-autoscaler.kubernetes.io/safe-to-evict",
        {},
```

Reply:
Thanks for noticing this. Fixed now.